stl/inc/xlocnum

```diff
@@ -657,9 +689,17 @@ private:
             }
         }
 
+        // skip leading separator before digits

```

Comment:
Nitpick: I believe this should say "separators" plural, because it has the same `while` loop.